frame/utility/src/lib.rs

```diff
@@ -459,6 +483,63 @@ pub mod pallet {
 			let base_weight = T::WeightInfo::batch(calls_len as u32);
 			Ok(Some(base_weight + weight).into())
 		}
+
+		/// Forwards a pre-signed function call.
+		///
+		/// # <weight>
+		/// - O(1).
+		/// - Limited storage reads.
+		/// - One DB write (event).
+		/// - Weight of derivative `call` execution + T::WeightInfo::dispatch_as().
+		/// # </weight>
```

Comment:
delete this stuff, and add a bit more documentation about the parameters of the call below and when / how to use this function

Reply:
done